app/components/BusinessBar.tsx

```diff
@@ -0,0 +1,82 @@
+"use client";
+import { BusinessCard } from "./BusinessCard";
+import { useState, FunctionComponent, useEffect } from "react";
+import supabase from "../config/supabaseClient";
+
+const HEADER_STYLING =
+    "header text-xl md:text-xl h-12 w-screen md:h-14 pl-5 py-2 mb-4 border-b border-border/70 sticky top-0 bg-background shadow-sm tracking-wide text-white";
+
+type BusinessType = {
+    id: number;
+    name: string;
+    rating: number;
+    reviewCount: number;
+    services: string[];
+    address: string;
+    createdAt: string;
+    imgSrc: string;
+};
+
+export const BusinessBar: FunctionComponent = () => {
+    const [businesses, setBusinesses] = useState<any | null>([]);
+
+    useEffect(() => {
+        const fetchBusinesses = async () => {
+            const { data, error } = await supabase.from("businesses").select();
+            if (data) {
+                setBusinesses(data);
+            }
+            if (error) {
+                throw error;
+            }
+        };
+        fetchBusinesses();
+    }, []);
+
+    const recommended = businesses.filter(
+        (business: BusinessType) => business.rating > 4.0
+    );
```

Comment:
You should add your slice here in your const. You can just chain it after your filter, it makes it cleaner. Generally you want to make all the modifications to the list before you map it out that way you're not unnecessarily rendering the items and then cropping them the way you have it currently.
```suggestion
    ).slice(0,6);
```

Reply:
Ah ok, gotcha.